WPMedia/fusion-news-theme-blocks
# TMEDIA-150 - Bandito update

## Description

The link in the ticket was to the 2.5 Engine docs which had an example of registerSuccessEvent that's not valid. This updates the shared components to use the correct syntax

2.5 
https://corecomponents.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/recipes/content-testing.md?version=2.5

```
import { registerSuccessEvent } from 'fusion:context'
```

2.7
https://corecomponents.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/recipes/content-testing.md?version=2.7

```
  const { registerSuccessEvent } = useComponentContext()
```


## Jira Ticket
- [TMEDIA-150](https://arcpublishing.atlassian.net/browse/TMEDIA-150)

## Acceptance Criteria
1. Customers can run Bandito tests on the following Arc Blocks: 
    * XL Promo Block
    * L Promo Block
    * Medium Promo Block
    * Small Promo Block
    * XL Manual Promo Block
    * L Manual Promo Block
    * Medium Manual Promo Block
    * Small Manual Promo Block

## Test Steps

1. Checkout this branch `git checkout TMEDIA-150-add-banito-to-promos`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/shared-styles`
3.  Verify promo blocks didn't change - test page - http://localhost/pf/test-all-promos/?_website=the-gazette


## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._
